mrmustard/math/lattice/steps.py

```diff
@@ -80,25 +80,24 @@ def vanilla_step_batch(
     The necessary pivot and neighbours must have already been computed,
     as this step will read those values from G.
     Note that this function is different from vanilla_step with b is no longer a vector,
-    it becomes a bathced vector with the batch dimension on the last index.
+    it becomes a bathced vector with the batch dimension on the first index.
 
     Args:
         G (array or dict): fock amplitudes data store that supports getitem[tuple[int, ...]]
```

Comment:
Unrelated to this specific PR but types arent really necessary in the docstring as long as the args are typehinted so these should probably be removed eg `G (array or dict)` -> `G`

Reply:
This seems to be a general thing for all functions in this file. I am not sure this is the PR to correct these things, but I agree with you.

Reply:
Yeah we probably just need a cleanup PR for these things